intellij/src/saros/core/ui/eventhandler/NegotiationHandler.java

```diff
@@ -215,11 +213,7 @@ protected IStatus run(IProgressMonitor monitor) {
                 MessageFormat.format(
                     Messages.NegotiationHandler_project_sharing_canceled_message, peerName);
 
-            ApplicationManager.getApplication()
-                .invokeLater(
-                    () ->
-                        NotificationPanel.showInformation(
-                            message, "Project sharing canceled remotely"));
+            NotificationPanel.showInformation(message, "Project sharing canceled remotely");
```

Comment:
(Not important for this PR:) The constant string can be moved into the messages.properties.

Reply:
There is a lot of stuff in the UI that needs cleaning up.